src/olympia/api/paginator.py

```diff
@@ -21,6 +21,8 @@ def validate_number(self, number):
             raise PageNotAnInteger('That page number is not an integer')
         if number < 1:
             raise EmptyPage('That page number is less than 1')
+        elif number > 100000:
```

Comment:
This should be either `sys.max_int` or 2147483647 (2^^32 - 1) directly which is the limit of the used `integer` type in ElasticSearch. At least that's what the actual exception is all about.

Now that'd be the proper technical fix, I doubt that there'll ever be more than 100k pages unless you own a bulletin board with a "the longest thread of the world"-thread so I'm fine with the 100.000 limit. Let's just put this as a constant at the top of the file for better visibility.